src/main/resources/liquibase/202406130700-add-index-in-sku-product-tag.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+  <changeSet id="202406130700-1" author="karshah">
+    <comment>Create an index on sku for sku_product_tag</comment>
+    <createIndex indexName="sku_idx" tableName="sku_product_tag"
```

Comment:
Would you mind changing the name real quick?

```suggestion
    <createIndex indexName="sku_product_tag_sku_idx" tableName="sku_product_tag"
```

This makes the name stand out if you list all the indexes.  Otherwise "sku_idx" is pretty ambiguous unless you know exactly what table you're already dealing with.